modules/persistent-storage-local-removing-devices.adoc

```diff
@@ -24,33 +19,72 @@ Deleting a persistent volume that is still in use can result in data loss or cor
 
 .Procedure
 
-. Edit the previously created local volume to remove any unwanted disks.
+To delete LVs or LVSs:
```

Comment:
```suggestion
To delete LVs or LVSs, complete the following steps:
```
Lead-in sentences should be complete sentences, per the SSG: https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences